src/core_ocean/analysis_members/Registry_eddy_product_variables.xml

```diff
@@ -39,6 +39,12 @@
 		<var name="velocityMeridionalTimesTemperature" type="real" dimensions="nVertLevels nCells Time" units="m s^{-1} C"
 			description="cell-wise product of component of horizontal velocity in the northward direction and temperature"
 		/>
+                <var name="velocityZonalTimesTemperature_GM" type="real" dimensions="nVertLevels nCells Time" units="m s^{-1} C"
+                        description="cell-wise product of component of horizontal bolus velocity in the eastward direction and temperature"
+                />
+                <var name="velocityMeridionalTimesTemperature_GM" type="real" dimensions="nVertLevels nCells Time" units="m s^{-1} C"
+                        description="cell-wise product of component of horizontal bolus velocity in the northward direction and temperature"
+                />
```

Comment:
You also need changes to src/core_ocean/Registry.xml to add the GMbolusVelocityZonal and Meridional arrays.

Reply:
again, ignore here.